sources_test.go

```diff
@@ -76,6 +76,47 @@ func TestExec(t *testing.T) {
 	}
 }
 
+func TestExecAt(t *testing.T) {
+	t.Parallel()
+	wdir, err := os.Getwd()
+	if err != nil {
+		t.Error(err)
+	}
+	want := "Usage"
+	p := ExecAt(wdir, "go")
+	if p.Error() == nil {
```

Comment:
We're really just testing Exec here, aren't we—this is a straight copy of TestExec, but with the working directory parameter added. Since we already know Exec works, from its own test, and since ExecAt should use Exec (see previous comment), _this_ test only needs to test that the command runs in the specified working directory (and also to test the behaviour in various edge cases, such as 'directory doesn't exist', 'permission denied', and so on).